docs/Install-Guide/compiling-rpms.md

```diff
@@ -5,28 +5,28 @@ Creating rpm's of GlusterFS from git source is fairly easy, once you know the st
 
 RPMS can be compiled on at least the following OS's:
 
--   Red Hat Enterprise Linux 5, 6 (& 7 when available)
--   CentOS 5, 6 (& 7 when available)
--   Fedora 16-20
+- Red Hat Enterprise Linux 5, 6 (& 7 when available)
```

Comment:
can be removed

Reply:
Same as above. Content changes in separate PR.